### ISSUE TITLE
Allow users to report spam or abusive stories

### ISSUE THREAD

**Issue by alex-wenzel** [instedd/oasis#160]
Given the unfortunate political dynamics coloring people's experiences with COVID-19, especially in the USA, there is a potential for the stories function to be used to spread abusive messages or misinformation. Users should be able to report stories that contain abusive language or COVID-19-related misinformation.

**Comment by yus252** [comment]
We should add a flag to the user whose story has been reported and stop show the user's story on the map without deleting it. 

In the short term, we should manually review the requests. In the long term, we should build up a spam system to review stories by AI.

**Comment by alex-wenzel** [comment]
I agree. I was planning to write code that would invoke the same email sender used for anonymous contacts to send an email to some preconfigured email address, something like abuse@oasis.instedd.org.

**Comment by yus252** [comment]
#183
